tests/data/placebo/test_ecs_service_taggable/ecs.DescribeClusters_1.json

```diff
@@ -0,0 +1,77 @@
+{
+    "status_code": 200,
+    "data": {
+        "clusters": [
+            {
+                "clusterArn": "arn:aws:ecs:us-east-1:644160558196:cluster/test-cluster",
```

Comment:
three of these clusters are extraneous and can be removed from the recorded output.

Reply:
removed